CHANGES.md

```diff
@@ -8,6 +8,7 @@ Change Log
 
 ##### Fixes :wrench:
 Disabled HDR by default to improve visual quality in most standard use cases. Set `viewer.scene.highDynamicRange = true` to re-enable it. [#7966](https://github.com/AnalyticalGraphicsInc/cesium/issues/7966)
+* Fixed sandcastle example *"Time Dynamic Wheels"* to load in non-built versions. [#7983](https://github.com/AnalyticalGraphicsInc/cesium/issues/7983)
```

Comment:
You can remove this, we generally don't include Sandcastle-only changes in CHANGES (and this was only a problem on cesiumjs.org, not running locally).

Reply:
Thanks! Wasn't sure.